third_party/ibis/ibis_mssql/compiler.py

```diff
@@ -371,10 +371,29 @@ def _day_of_week_name(t, expr):
     (sa_arg,) = map(t.translate, expr.op().args)
     return sa.func.trim(sa.func.format(sa_arg, 'dddd'))
 
+
 def _string_join(t, expr):
     sep, elements = expr.op().args
     return sa.func.concat(*map(t.translate, elements))
 
+
+def _timestamp_truncate(t, op):
```

Comment:
Has this been tested? I'm not sure if this will work since the older versions of Ibis use different signatures i.e. function(t, expr) instead of function(t,ops).

I think we can stick to supporting ExtractEpochSeconds in this PR since the other methods aren't used in DVT

Reply:
Agreed, I didn't know how to test all of them but the ExtractEpochSeconds is working! Removed the others